justfile

```diff
@@ -348,6 +363,8 @@ rctemplate := '''
 container_registry=""
 # Azure resource group/ resource name. Resource group will be created.
 azure_resource_group=""
+# Azure subscription id.
+azure_subscription_id="0d202bbb-4fa7-4af8-8125-58c269a05435"
```

Comment:
This is in the wrong section, should go further down as it doesn't need to be changed.